analysis_driver/pipelines/qc_gatk4.py

```diff
@@ -107,10 +108,6 @@ def vqsr_snps_tranches(self):
     def vqsr_snps_r_script(self):
         return self.gatk4_basename + '_vqsr_snps.R'
 
-    @property
-    def vqsr_filtered_snps_vcf(self):
-        return self.gatk4_basename + '_vqsr_snps.vcf.gz'
```

Comment:
May I ask why this has been removed?

Reply:
This was an intermediate file name that was use as the output of Apply VQSR for SNP stage.
I'm  now running the VQSR for SNP and indels so I only need one intermediate file.